docs/UserGuide.md

```diff
@@ -414,6 +425,30 @@ _Details coming soon ..._
 1. Click the Launchpad icon in the Dock, type Terminal in the search field, then click Terminal.
 2. In the Finder, open the `/Applications/Utilities` folder, then double-click Terminal.
 
+<box type="info" seamless>
```

Comment:
hmm instead of the box shldnt it be like:
**Q**: How do you fill up the *role* and *material* information within the UI?
**A**: <ur answer>

Reply:
doneded